knowledge-base/how-to-register-sqldatasource-data-providers-without-driver-installation.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Registering SqlDataSource data providers without installing driver
+description: No need to install a driver when you register SQLDataSource. See how to register a .Net Framework data provider without installing its driver. 
+type: how-to
+page_title: Registering .Net Framework data provider without installing its driver
+slug: how-to-register-sqldatasource-data-providers-without-driver-installation
+res_type: kb
+category: knowledge-base
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Report Item</td>
+			<td>SqlDataSource Component</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+You need to register a .Net Framework data provider for Telerik Reports without installing its driver.

```

Comment:
Remove this sentence or reword it to a question. E.g. "How can I register a .Net Framework data provider for Telerik Reporting without installing its driver?"